Tools/machines/karolina-it4i/karolina_warpx.profile.example

```diff
@@ -0,0 +1,81 @@
+# please set your project account, ie DD-N-N
+export proj="<proj_id>"  # change me!
+
+# Name and Path of this Script ################### (DO NOT change!)
+export MY_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+if [ -z ${proj-} ]; then
+    echo "WARNING: The 'proj' variable is not yet set in your $MY_PROFILE file!"
+    echo "Please edit its line 2 to continue!"
+    return
+fi
+
+# set env variable storing the path to the work directory
+# please check if your project ID belongs to proj1, proj2, proj3 etc
+export WORK="/mnt/proj<N>/${proj,,}/${USER}"  # change me!
+mkdir -p WORK
+
+# clone warpx
+# you can also clone your own fork here, eg git@github.com:<user>/WarpX.git
+if [ ! -d "$WORK/src/warpx" ]
+then
+    git clone https://github.com/ECP-WarpX/WarpX.git $WORK/src/warpx
+fi
+
+# download and activate spack
+if [ ! -d "$WORK/spack" ]
+then
```

Comment:
Having this in an extra `install_..._dependencies.sh` script is intentional:
- it allows you to clean up prior installs
- it gives the user control to wait hours for the first install (`spack install`) when they are ready